ui/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,9 @@
     <uses-feature
         android:name="android.hardware.camera.any"
         android:required="false" />
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
```

Comment:
Is this needed because `android.hardware.camera.any` came in a later Android release than our minSdk? Can you please document that version in a comment here if so?